snippets/chain-async-functions.md

```diff
@@ -0,0 +1,40 @@
+### Chain asynchronous functions
+
+Loop through an array of functions containing asynchronous events, calling `next` when each asynchronous event has completed.
+
+```js
+const chainAsync = fns => {
+	let curr = 0; const next = () => fns[curr++](next); next()
+}
+```
+
+<details>
```

Comment:
Indeed `<details>` is a pretty cool way to explain, but it doesn't play on any MS browser, so this is probably a bit of a problem. Maybe linking to an explanation or article would be preferrable.

Reply:
I wasn't aware of the MS limitation. Shouldn't be surprised. Thanks.